doc/source/groupby.rst

```diff
@@ -1301,11 +1315,15 @@ Piping can also be expressive when you want to deliver a grouped object to some
 arbitrary function, for example:
 
 .. code-block:: python
+   def mean(groupby):
+       return(groupby.mean())
```

Comment:
not sure why flake8 doesn't complain here, but in Python the correct way is `return groupby.mean()` without the brackets.

Reply:
@datapythonista I removed the brackets, see 00b03f0c5f2e1d05f1da98faafb37ab0575f3fcd